### PR TITLE
Buidingview addtask

### PR DESCRIPTION
**Pull request by thientran98qb**
ready

### CODE REVIEW

**Comment by thientran98qb** [comment]
ready

File: resources/views/common/errors.blade.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@if (count($errors) > 0)
+    <!-- Form Error List -->
+    <div class="alert alert-danger">
+        <strong>Whoops! Something went wrong!</strong>
```
Comment (quoctd-0607) [inline resources/views/common/errors.blade.php]: i18n

File: resources/views/tasks/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                            <div class="col-sm-6">
+                                <input type="text" name="name" id="task-name" class="form-control">
+                            </div>
+                        </div>
+            
+                        <!-- Add Task Button -->
+                        <div class="form-group">
+                            <div class="col-sm-offset-3 col-sm-6">
+                                <button type="submit" class="btn btn-info">
+                                    <i class="fa fa-plus"></i> Add Task
```
Comment (quoctd-0607) [inline resources/views/tasks/index.blade.php]: tương tự

**Comment by thientran98qb** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**